### PR TITLE
WIP: 2.0

### PR DESCRIPTION
**Pull request by connor-baer**
# DONT MERGE THIS

This branch is a **work in progress** of the new brand design that will be used starting with the new [component library](https://github.com/mlent/component-library). 
I've also updated some dependencies, most notably Bourbon to 5.0 and Neat to 2.0. Have a look at the [Bourbon 5.0](http://bourbon.netlify.com/docs/latest/) and [Neat 2.0](http://neat.bourbon.io/docs/2.0.0/) docs.

### TODO

- [x] Update to Bourbon 5.0 and Neat 2.0 and fix deprecation warnings
- [x] Update to new design based on the [fabulous work](https://share.abstractapp.com/64ada0c7-9069-4f7a-b632-1204dbc8e739) of the design team
- [x] Lint using [stylelint](https://stylelint.io)
- [x] Update README
- [ ] Include updates from `master`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,16 +9,16 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/felixjung/brand-styles.git"
+    "url": "git+https://github.com/sumup/brand-styles.git"
```
Comment (mollidor) [inline package.json]: 😄

Comment (felixjung) [reply package.json]: 😆

File: _variables.scss [hunk 2]
```diff
@@ -89,6 +89,15 @@ $breakpoints: (
   large: 93.75em   // 1500px
 );
 
+// TODO:👇 Deprecated (in use on dashboard)
+$until-medium-screens: new-breakpoint(max-width 450px 4) !default;
```
Comment (felixjung) [inline _variables.scss]: Yo, FYI, dashboard has it's own bad breakpoints 😁 These are only used in static-marketing.

Comment (connor-baer) [reply _variables.scss]: Thanks!

**Comment by connor-baer** [comment]
Closing, since we've moved to [⚡Circuit UI](https://github.com/sumup/circuit-ui) since.

**Pull request closed without merging**